src/QualificationRunner.Core/Services/QualificationEngine.cs

```diff
@@ -66,22 +66,34 @@ private async Task<QualificationRunResult> execute(QualificationConfiguration qu
 
          _logger.AddDebug(Logs.QualificationConfigurationForProjectExportedTo(project, configFile));
 
-         var cliPath = qualificationConfiguration.Application == ApplicationType.PKSim
-            ? _applicationConfiguration.PKSimCLIPathFor(runOptions.PKSimInstallationFolder)
-            : _applicationConfiguration.MoBiCLIPathFor(runOptions.MoBiInstallationFolder);
+         string cliPath, moBiPKSimStarterPath = string.Empty;
+         if (qualificationConfiguration.Application == ApplicationType.PKSim)
+            cliPath = _applicationConfiguration.PKSimCLIPathFor(runOptions.PKSimInstallationFolder);
+         else
+         {
+
+            cliPath = _applicationConfiguration.MoBiCLIPathFor(runOptions.MoBiInstallationFolder);
+            
+            // If the PK-Sim folder was specified by command line argument the intent is to inform MoBi which PK-Sim instance
+            // should be used for PK-Sim services.
+            if (!string.IsNullOrEmpty(runOptions.PKSimInstallationFolder))
+               moBiPKSimStarterPath = Path.Combine(runOptions.PKSimInstallationFolder, Constants.Tools.PKSIM);

```

Comment:
shouldn't it be the PK-Sim CLI path exactly as determined above for PK-Sim? so 
`moBiPKSimStarterPath = _applicationConfiguration.PKSimCLIPathFor(runOptions.PKSimInstallationFolder);`

Reply:
No, because we don't use the CLI path to run the PK-Sim starter.